examples/pipelines/ccfraud/submit.py

```diff
@@ -181,7 +187,7 @@ def getUniqueIdentifier(length=8):
 @pipeline(
     description=f'FL cross-silo basic pipeline and the unique identifier is "{pipeline_identifier}" that can help you to track files in the storage account.',
 )
-def fl_cross_silo_internal_basic():
+def fl_ccfraud_basic():
```

Comment:
Why to rename function in every example to be custom one? Isn't it better to name them same way so we have some consistency?

Reply:
The name of the function is used (currently) as a run name. So my intention here is we have 3 different experiments, one for each scenario, and in each experiment, we have runs with name aligning with the scenario as well so that when you look at "all runs" in the workspace, you see which one is which.